server/plugin/command.go

```diff
@@ -87,10 +92,69 @@ func azureDevopsAccountConnectionCheck(p *Plugin, c *plugin.Context, commandArgs
 	if isConnected := p.UserAlreadyConnected(commandArgs.UserId); !isConnected {
 		return p.sendEphemeralPostForCommand(commandArgs, p.getConnectAccountFirstMessage())
 	}
-
+	if len(args) > 0 {
+		switch {
+		case args[0] == "subscriptions":
+			return azureDevopsSubscribeCommand(p, c, commandArgs, args...)
+		case args[0] == "unsubscribe":
+			return azureDevopsUnsubscribeCommand(p, c, commandArgs, args...)
+		}
```

Comment:
do we need a default case here if args[0] is neither subscriptions nor subscriptions

Reply:
no, that is laready handled and we are showing `Invalid command parameters. Please use /azuredevops help for more information.`